frontend/benefit/applicant/src/components/applications/pageContent/PageContent.tsx

```diff
@@ -46,6 +59,55 @@ const PageContent: React.FC = () => {
     );
   }
 
+  // if view mode, show customized summary
+  if (
+    application.status &&
+    SUBMITTED_STATUSES.includes(application.status) &&
+    Boolean(isReadOnly)
+  ) {
+    return (
+      <Container>
+        <$PageHeader>
+          <$HeaderItem>
+            <$PageHeading>
+              {t('common:applications.pageHeaders.edit')}
```

Comment:
`edit` in readonly mode?

Reply:
well, its the same contents, in both edit and view its Hakemus translation, i just didnt duplicate